hack/update-bazel.sh

```diff
@@ -22,16 +22,15 @@ KOPS_ROOT=$(git rev-parse --show-toplevel)
 cd ${KOPS_ROOT}
 TMP_GOPATH=$(mktemp -d)
 
+ln -s ${KOPS_ROOT}/vendor/ ${TMP_GOPATH}/src
+
+GOPATH=${TMP_GOPATH}/ go install github.com/bazelbuild/bazel-gazelle/cmd/gazelle
```

Comment:
rather than creating a fresh GOPATH, could you use `go build -o` to write the gazelle binary into a tempdir? that way you'd hopefully get caching of most of the go packages each time this is run?